### PR TITLE
[#865] Add Aria Label to Search Button

### LINKED ISSUE #865: Search Button Does Not Have Accessible Name
The Search button in [frontpage.html](https://github.com/inveniosoftware/invenio-communities/blob/master/invenio_communities/templates/semantic-ui/invenio_communities/frontpage.html#L39) does not have an accessible name. 
https://dequeuniversity.com/rules/axe/4.4/button-name?utm_source=lighthouse&utm_medium=devtools

To fix, add `aria-label` to the button element.

### PR DESCRIPTION
**Pull request by sharpattack**
Search button needs an accesible name. fixes #865

:heart: Thank you for your contribution!

### Description

Please describe briefly your pull request.

### Checklist

Ticks in all boxes and 🟢 on all GitHub actions status checks are required to merge:

- [x] I'm aware of the [code of conduct](https://inveniordm.docs.cern.ch/contribute/code-of-conduct/).
- [x] I've created [logical separate commits](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commits) and followed the [commit message format](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commit-message).
- [ ] I've added relevant test cases.
- [ ] I've added relevant documentation.
- [ ] I've marked [translation strings](https://inveniordm.docs.cern.ch/develop/best-practices/i18n/) (for relevant code).
- [ ] I've followed the [CSS/JS](https://inveniordm.docs.cern.ch/develop/best-practices/css-js/) and [React](https://inveniordm.docs.cern.ch/develop/best-practices/react/) guidelines (for relevant code).
- [x] I've followed the [web accessibility](https://inveniordm.docs.cern.ch/develop/best-practices/accessibility/) guidelines (for relevant code).
- [ ] I've followed the [user interface](https://inveniordm.docs.cern.ch/develop/best-practices/ui/) guidelines (for relevant code).
- [x] I've identified the [copyright holder(s)](https://inveniordm.docs.cern.ch/contribute/copyright-policy/) and updated copyright headers for touched files (>15 lines contributions).
- [x] I've NOT included third-party code (copy/pasted source code or new dependencies).

**Third-party code**

If you've added [third-party code (copy/pasted or new dependencies)](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#third-party-codedependencies), please reach out to an [architect](https://github.com/orgs/inveniosoftware/teams/architects/members).

**Reminder**

By using GitHub, you have already agreed to the [GitHub’s Terms of Service](https://help.github.com/articles/github-terms-of-service/#6-contributions-under-repository-license) including that:

1. You license your contribution under the same terms as the current repository’s license.
2. You agree that you have the right to license your contribution under the current repository’s license.

### CODE REVIEW

**Comment by fenekku** [comment]
Ah I see, I think you have to be a member of the project to be able to assign reviewers. I've assigned the ones above :+1:

**Review by kpsherva — APPROVED** [review]

File: invenio_communities/templates/semantic-ui/invenio_communities/frontpage.html [hunk 1]
```diff
@@ -36,7 +37,7 @@ <h1 class="ui header">
         <form action="{{ url_for('invenio_communities.communities_search') }}" class="ui form">
           <div class="ui fluid action input">
             <input type="text" name="q" class="form-control" placeholder="{{ _('Search communities') }}">
-            <button type="submit" class="ui icon search button"><i class="search icon"></i></button>
+            <button type="submit" class="ui icon search button" aria-label="Search communities"><i class="search icon"></i></button>
```
Comment (fenekku) [inline invenio_communities/templates/semantic-ui/invenio_communities/frontpage.html:40]: Oh just noticed this: at NU we can get away without translations, but for international partners it's probably nicer to have accessible label translated:

```suggestion
            <button type="submit" class="ui icon search button" aria-label="{{ _('Search communities'} }}"><i class="search icon"></i></button>
```

Or given the line is pretty long :sweat_smile: :

```suggestion
            <button 
              type="submit" 
              class="ui icon search button" 
              aria-label="{{ _('Search communities') }}"
            >
              <i class="search icon"></i>
            </button>
```

**Review by fenekku — APPROVED** [review]

**Comment by sharpattack** [comment]
Thank for your reviews and approvals @kpsherva and @fenekku! I committed Guillaume's suggestion. 

As I was trying to re-request your reviews, github was acting strange. I did not mean to perform all of the reported actions. Karolina and Guillaume, please re-review and re-assign me. Nothing else. Thank you!

File: invenio_communities/templates/semantic-ui/invenio_communities/frontpage.html [hunk 2]
```diff
@@ -36,7 +37,13 @@ <h1 class="ui header">
         <form action="{{ url_for('invenio_communities.communities_search') }}" class="ui form">
           <div class="ui fluid action input">
             <input type="text" name="q" class="form-control" placeholder="{{ _('Search communities') }}">
-            <button type="submit" class="ui icon search button"><i class="search icon"></i></button>
+            <button 
+              type="submit" 
+              class="ui icon search button" 
+              aria-label="{{ _('Search communities') }}"
+            >
+              <i class="search icon"></i>
```
Comment (jennur) [inline invenio_communities/templates/semantic-ui/invenio_communities/frontpage.html:45]:
```suggestion
              <i class="search icon" aria-hidden="true"></i>
```

**Review by jennur — APPROVED** [review]
Just a small comment :)

File: invenio_communities/templates/semantic-ui/invenio_communities/frontpage.html (hunk 2 above)
Comment (sharpattack) [reply invenio_communities/templates/semantic-ui/invenio_communities/frontpage.html]: Thank you!

**Comment by fenekku** [comment]
Thanks @sharpattack !

**Pull request merged**